pkg/util/constants.go

```diff
@@ -11,6 +11,7 @@ const (
 	AnnotationTimestamp            = prefix + "/timestamp"
 	AnnotationVolumeClaimTemplates = prefix + "/volumeClaimTemplates"
 	AnnotationImageID              = prefix + "/imageId"
+	AnnotationReservedMemory       = prefix + "/reservedMemory"
```

Comment:
As discussed with @yasker before, he preferred the annotations to be with format of `reserved-memory`

Reply:
It seems the recommendation from K8s is `camel-case`, though I will leave it up to the engineering team to decide. e.g. https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
We can file a new issue to track those types of issues, for now, we can keep it for better consistency.